### ISSUE TITLE
Don't you think an extension is a better solution?

### ISSUE THREAD

**Issue by ghost** [rakibtg/BigTube#1]
Copy and paste a link to another site is troublesome. Don't you think a browser extension for Google Chrome is a better solution (Although it might not that kind of flexible in term of cross browsers)?

**Comment by murshid1988** [comment]
I agree with @li-xinyang. This will be still a pain, since you will have to search for the video in youtube and copy paste the url back here.

**Comment by mnemanja** [comment]
Check this extension out. I've been using it a lot https://www.chromeactions.com/magic-options.html

**Comment by purplematter** [comment]
maybe just transform the url? https://www.youtube.com/watch?v=fOyP3Y1N2Lk becomes
https://www.youtube.com/v/fOyP3Y1N2Lk
